external/upstream/libint/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+if(${ENABLE_Libint1t})
+find_package(Libint 1.2.0 CONFIG QUIET COMPONENTS ${MAX_AM_ERI})
```

Comment:
Hard no to bringing Libint 1 back. Did you mean to commit this? Based on your PR description, you shouldn't need any of these cmake changes.

Reply:
No worries -- this was the result of a hackathon with Konrad Patkowski, and there's something wrong with the origin -- we'll get it fixed, just wanted it public.